Package.resolved

```diff
@@ -15,8 +15,8 @@
         "repositoryURL": "https://github.com/DataDome/datadome-ios-package",
         "state": {
           "branch": null,
-          "revision": "e6decadef472aa1d978f9fad158d689441978769",
-          "version": "2.8.0"
+          "revision": "c348ce0463494c224adf540f139dfb8b6c546333",
+          "version": "3.0.1"
```

Comment:
Wrong version?

Reply:
That's the resolved file, it doesn't matter.
Only Package.swift define the dependencies of our Package

Reply:
I removed the file from revision